packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -1685,16 +1693,20 @@ async function withController<ReturnValue>(
     updateIdentities: sinon.stub(),
     setSelectedAddress: sinon.stub(),
   };
-  const controller = new KeyringController(preferences, {
+  const messenger = buildKeyringControllerMessenger();
+  const controller = new KeyringController({
     encryptor,
     cacheEncryptionKey: false,
+    messenger,
+    ...preferences,
     ...rest,
   });
-  const initialState = await controller.createNewVaultAndKeychain(password);
```

Comment:
As we removed `vault`, `encryptionKey` and `encryptionSalt` from methods returns, we now need to take the initial state from `controller.state`, which contains all state properties